diagnostic.md

```diff
@@ -32,22 +39,25 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
+$("text")
+$("value")
```

Comment:
A couple of examples are `.val()` and `.html()`.  Both of these methods are getters (without arguments) and setters (with argument(s)).